src/core/polyhedral/cuda/mapped_scop.cc

```diff
@@ -329,10 +316,8 @@ detail::ScheduleTree* MappedScop::separateReduction(detail::ScheduleTree* st) {
 size_t MappedScop::mapToThreads(detail::ScheduleTree* band, size_t nInner) {
   using namespace tc::polyhedral::detail;
 
-  if (nInner >= numThreads.view.size()) {
-    return nInner;
-  }
```

Comment:
Why did you remove this test?
In fact, it should be strengthened to `nInner > 0`, no?

Reply:
`nInner` is completely removed in the following commit, I may have picked the wrong change